source/documentation/runbooks/004-analytical-platform-compute-maintenance.html.md.erb

```diff
@@ -0,0 +1,116 @@
+
+
+On the first day of the month the workflow [schedule-issue-compute-infrastructure.yml] (https://github.com/ministryofjustice/analytical-platform/blob/bba07f4908b443a2c15bf94738c8b84c5211e9f9/.github/workflows/schedule-issue-compute-infrastructure.yml) will automatically raise a ticket for [Maintenance - Analytical Platform Compute ] (https://github.com/ministryofjustice/analytical-platform/issues/5661 ).
+
+This maintenance ticket includes EKS Cluster Upgrade if applicable and/or patching to ensure all components are up to date.
+
+
+##  Check for new release
+
+Check if a new release of Amazon EKS Kubernetes has been made available [here](https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html#kubernetes-release-calendar
+).
+
+If there is a new EKS release available carry out EKS upgrade and patch all other components.
+
+Or
+
+If there is not a new EKS release available patch all other components including addons.
+
+## The Approach
+
+1. Create a new branch / PR.
+1. Make changes to the code.
+1. Create a Pull Request and check the plan.
+1. Request approval and merge the PR.
+
+### Order
+
+Apply in the development, test and production environments, resolving any issues before progressing to the next.
+
+## Working on APC in Modernisation platform
+
+Be aware that due to restrictions with the state file multiple people cannot work on the environment at once. Terraform plan is fine but once you have carried out a local apply the state file cannot be used by anyone until the changes are merged.
+
+Make the team aware and check before starting the work to avoid conflicts.
+
+## Assumptions
+
+- You are operating in the `modernisation-platform-environments` repository Development Container.
+- To interrogate the cluster, you are exec'd into the same account as the cluster you are operating on `aws-sso exec --profile analytical-platform-compute-test:modernisation-platform-developer`.
+- Use account `modernisation-platform-developer` for Test and Procuction and `modernisation-platform-sandbox` for Development.
+- If necessary update `~/kube/config` as follows `aws eks update-kubeconfig --region eu-west-2 --name analytical-platform-compute-test`.
+- Set context as follows  `kubectl config use-context arn:aws:eks:eu-west-2:767397661611:cluster/analytical-platform-compute-test`.
+
+
+**Note:** amend above appropriately for the environment you are working in.
+
+## Impact on Users
+
+As this is a live service there could be an impact on users so this will have to be taken into consideration when planning the work.
+
+The impact on users depends on what is planned to be upgraded/patched.
+
+For example
+If upgrading `cloudwatch logs` agent, the user impact is minimal, applications will run, logs might be delayed and you will not require a maintenance window.
+
+IF upgrading `karpenter`, the user impact is high because jobs might not schedule as expected so you will have to agree when to schedule a maintenance window.
```

Comment:
``` suggestion
If upgrading [karpenter](https://karpenter.sh/), the user impact is potentially higher because jobs might not schedule as expected so you will have to agree when to schedule a maintenance window.
```

Reply:
amended